content-services/content-sources-backend
# Fixes 3424: add label to repository configs

## Summary

Adds a label column to the repository configurations table, and returns it from the repositories api.  
For custom repos this will currently be blank (in the future we will likely set this).
For red hat repos, this will be populated
For the snapshot config.repo api, this label is used (if present). 

## Testing steps

* check out main and run 'make repos-import'.  GET /repositories?origin=external,red_hat and see that there is no label attribute
* checkout this pr, db migrate, run `make repos-import` again
* refetch the repositories via the same api and confirm that there is now a label.
* Create a custom repo and see that its label is blank
* trigger a snapshot of a redhat repo (go run cmd/external-repos/main.go snapshot $URL)  
* fetch the config.repo file and confirm that the correct label is there:   GET /repositories/UUID/snapshots/UUID/config.repo

## Checklist
